dataclients/kubernetes/kube.go

```diff
@@ -363,11 +364,14 @@ func mapRoutes(r []*eskip.Route) map[string]*eskip.Route {
 }
 
 func (c *Client) loadAndConvert() ([]*eskip.Route, error) {
+	c.mu.Lock()
```

Comment:
Why not defer?

Reply:
Because the function does too much and I want to reduce holding the lock